.travis.yml

```diff
@@ -31,5 +31,4 @@ env:
   - TOXENV=pep8
   - TOXENV=check_commit_msg
   - TOXENV=py27-cover-master
-  - TOXENV=py27-cover-develop
```

Comment:
This change is not as part of my PR. While pushing the PR i updated with latest master, but the spec branch is old and outdated. Hence all the commits that was added during my rebase are appearing as new commits.
I guess if some maintainer rebases the spec branch the issue will be resolved